test/testing/run.spec.ts

```diff
@@ -711,6 +717,130 @@ describe('Testing SDK', () => {
     });
   });
 
+  it('collects test events', async () => {
+    const timestamp = new Date().toISOString();
+
+    class MyEvaluator1 extends BaseTestEvaluator<MyTestCase, string> {
+      id = 'my-evaluator-1';
+
+      evaluateTestCase(): Evaluation {
+        return { score: 0.5 };
+      }
+    }
+
+    const sleep = (ms: number) =>
+      new Promise((resolve) => setTimeout(resolve, ms));
+
+    await runTestSuite<MyTestCase, string>({
+      id: 'my-test-id',
+      testCases: [
+        { x: 1, y: 2 },
+        { x: 3, y: 4 },
+      ],
+      testCaseHash: ['x', 'y'],
+      evaluators: [new MyEvaluator1()],
+      fn: async ({ testCase }: { testCase: MyTestCase }) => {
+        const tracer = new AutoblocksTracer({
+          ingestionKey: 'test',
+          timeout: {
+            milliseconds: 1000,
+          },
+        });
+        // Simulate one doing more work than the other to make sure concurrency is handled correctly
+        if (testCase.x === 1) {
+          await sleep(500);
+        }
+        const message = `${testCase.x} + ${testCase.y} = ${testCase.x + testCase.y}`;
+        await tracer.sendEvent(message, {
+          timestamp,
+          traceId: 'test-trace-id',
+          properties: {},
+        });
+        return message;
+      },
+      maxTestCaseConcurrency: 2,
```

Comment:
do you wanna add the same comment about how this needs to be equal to the number of test cases to test concurrency properly